README.md

```diff
@@ -5,13 +5,19 @@ The CiviAwards extension depends on (uk.co.compucorp.civicase) extension and pro
 and processes.
 
 ## How do I get CiviAwards?
-CiviAwards is designed to work with CiviCRM 5.x plus. If you are on an earlier version of CiviCRM, you will need to upgrade your site first or contact info@compucorp.co.uk if you needs assistance to do so.
 
-If your CiviCRM is already on CiviCRM 5.x plus and this is the first time you use an extension,  please see [Here](http://wiki.civicrm.org/confluence/display/CRMDOC/Extensions "CiviCRM Extensions Installation") for full instructions and information on how to set and configure extensions.
+### Technical Requirements
+* [CiviCase](https://github.com/compucorp/uk.co.compucorp.civicase/releases): version >= v1.0.0-beta.5, preferably the latest version.
+
+### Installation
+If this is the first time you use an CiviCRM extension,  please see [Here](http://wiki.civicrm.org/confluence/display/CRMDOC/Extensions "CiviCRM Extensions Installation") for full instructions and information on how to set and configure extensions.
 
 You can get the latest release of CiviAwards from [Github repository release page](https://github.com/compucorp/uk.co.compucorp.civiawards/releases).
 
+Contact info@compucorp.co.uk if you needs assistance to do so
+
+
 ## Support
 Please contact the follow email if you have any question: <hello@compucorp.co.uk>
```

Comment:
Should be `Please contact the following email if you have any question: <hello@compucorp.co.uk>`
Do we need to use separate emails for support and for contact? i.e info@/hello@

Reply:
I think the 2 emails are different, `info@compucorp.co.uk` is for normal information. But `hello@compucorp.co.uk` is for paid support(as described in the readme).